src/components/v2/appDetails/k8Resource/nodeType/Node.component.tsx

```diff
@@ -138,6 +138,24 @@ function NodeComponent({
         }
     }, [params.nodeType, podType, url, filteredNodes]);
 
+    const getElapsedTime = (createdAt : Date) => {
+        let currentDate = new Date()
```

Comment:
As discussed, please update this function to be something simpler & more readable.